sunpy/util/decorators.py

```diff
@@ -221,3 +223,43 @@ def __call__(self, func):
         if self.kwargs:
             func.__doc__ = func.__doc__.format(**self.kwargs)
         return func
+
+def deprecate_positional_args(f):
+    """Decorator for methods that issues warnings for positional arguments
```

Comment:
```suggestion
    """
    Decorator for methods that issues warnings for positional arguments
```
I know this was copied.